Sources/LeafKit/LeafRenderer.swift

```diff
@@ -17,10 +17,20 @@ public protocol LeafCache {
         _ document: ResolvedDocument,
         on loop: EventLoop
     ) -> EventLoopFuture<ResolvedDocument>
+	
     func load(
         documentName: String,
         on loop: EventLoop
     ) -> EventLoopFuture<ResolvedDocument?>
+	
+    /// - return nil if cache entry didn't exist in the first place, true if purged
+    /// - will never return false in this design but should be capable of it
+    ///   in the event a cache implements dependency tracking between templates
+	@discardableResult func purge(
```

Comment:
Futures shouldn't be `@discardableResult`.

Reply:
What about the name `remove`? That's an antonym for insert.

Reply:
There will need to be a default implementation for this method so that this isn't a breaking change.

Reply:
D'oh. I did miss that note. Any objection to the default implementation just fatalError'ing since it should be unreachable in use?